packages/aws-cdk/lib/api/cloudformation-deployments.ts

```diff
@@ -1,10 +1,14 @@
+import * as path from 'path';
 import * as cxapi from '@aws-cdk/cx-api';
 import { AssetManifest } from 'cdk-assets';
+import * as fs from 'fs-extra';
+import { LazyListStackResources, ListStackResources } from '../api/evaluate-cloudformation-template';
 import { Tag } from '../cdk-toolkit';
 import { debug, warning } from '../logging';
 import { publishAssets } from '../util/asset-publishing';
 import { Mode, SdkProvider, ISDK } from './aws-auth';
 import { deployStack, DeployStackResult, destroyStack } from './deploy-stack';
+import { EvaluateCloudFormationTemplate } from './hotswap/evaluate-cloudformation-template';
```

Comment:
This is wrong, no? The class `EvaluateCloudFormationTemplate` has been moved to `lib/api/evaluate-cloudformation-template.ts`.

The above new import for `(Lazy)LazyListStackResources` is correct.